src/libstd/collections/hash/map.rs

```diff
@@ -1273,6 +1273,13 @@ impl<'a, K, V> Clone for Iter<'a, K, V> {
     }
 }
 
+#[stable(feature = "std-debug", since = "")]
+impl<'a, K, V> fmt::Debug for Iter<'a, K, V> {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        f.pad("Iter {{ .. }}")
```

Comment:
Oh I managed to forget that this is an iterator and maybe it should have a new or different style of display.. I don't know.